src/kw_ssh.sh

```diff
@@ -39,7 +39,9 @@ function kw_ssh_main()
 
   script_path=${options_values['SCRIPT']}
   cmd=${options_values['CMD']}
-  flag=${options_values['VERBOSE']}
+
+  [[ -n "${options_values['VERBOSE']}" ]] && flag='VERBOSE'
+  flag=${flag:-'SILENT'}
```

Comment:
These lines, together with the `parser_ssh_options` are the root cause of your test failures. For example, try this:

```diff
 
-  [[ -n "${options_values['VERBOSE']}" ]] && flag='VERBOSE'
   flag=${flag:-'SILENT'}
+  [[ -n "${options_values['VERBOSE']}" ]] && flag='VERBOSE'
+  [[ "${options_values['VERBOSE']}" -eq 'TEST_MODE' ]] && flag='TEST_MODE'
```

Please, review your code carefully and do not use the above suggestion in your final PR since this is a workaround to highlight the issue.